### DISCUSSION TITLE
Suggesting features

### DISCUSSION THREAD

**Discussion by ghost** [eusonlito/Password-Manager#39] [Ideas]
Hello,

I'm currently testing Password-Manager and I'd like to suggest some nice to have features.
- The administrator should be able to export Shared Applications by tags (for backup purposes for instance) 
- Folders and sub-folders : Could be useful if for instance you want to sort the credentials in some kind of tree (eg : by Customer Country -> Customer Name -> PROD/TEST/DEV)

**Comment by eusonlito** [comment]
Yes, first point is easy to add. Folder and subfolder will change all UI is not easy.

**Comment by eusonlito** [comment]
- [x] Added Export Shared Apps option to Admins.

**Reply by ghost** [reply]
Suggesting another greeatly missing feature :  Attaching files to any App type.

**Reply by eusonlito** [reply]
> Suggesting another greeatly missing feature : Attaching files to any App type.

Moved to issue :)
